openpype/hosts/aftereffects/api/pipeline.py

```diff
@@ -37,7 +38,7 @@ def check_inventory():
     if not lib.any_outdated():
         return
 
-    host = pyblish.api.registered_host()
+    host = registered_host()
     outdated_containers = []
     for container in host.ls():
         representation = container['representation']
```

Comment:
I know it's not related to this PR. But this logic is better of just calling [`any_outdated()`](https://github.com/pypeclub/OpenPype/blob/develop/openpype/lib/avalon_context.py#L161-L188) to ensure unified behavior across all hosts.

Reply:
I didn't see it's called on line 37 :D

Reply:
Haha, damn it! :) You are right - missed that too.